packages/vx-demo/src/sandboxes/vx-chart-poc/src/hooks/useRegisteredData.ts

```diff
@@ -0,0 +1,13 @@
+import { useContext } from 'react';
+import ChartContext from '../context/ChartContext';
+import { DataRegistry } from '../types';
+
+export default function useRegisteredData<
```

Comment:
one consequence of the `context` approach is that the `data`/`xAccessor`/`yAccessor passed to a `*Series` may be out of sync with the `xScale`/`yScale` from context. Getting these values from context prevents this, but is an extra step in the implementation of a `*Series`